HelpSource/Classes/Dictionary.schelp

```diff
@@ -193,24 +193,31 @@ d.matchAt(\abc)
 ::
 
 method::trueAt
-Returns link::Classes/True:: if the item's booleanValue at the key is true, otherwise false. This method is also valid in link::Classes/Object::.
-code::
-d = (x:true, y:false);
-d.trueAt(\x) // true
-d.trueAt(\y) // false
-d.trueAt(\foo) // false
+Returns link::Classes/True:: if the item's teletype::booleanValue:: at the key is teletype::true::, otherwise teletype::false::.
 
+code::
 // using binary value equivalents
 d = (x:1, y:0);
-d.trueAt(\x) // true
-d.trueAt(\y) // false
+d.trueAt(\x)   // true
+d.trueAt(\y)   // false
 d.trueAt(\foo) // false
+
+// other kinds of objects
+d = (num: 23.7, yes:true, no:false, sym:\foo);
```

Comment:
23.7 is not really the intended use. It should be said that this will be true, but `1` and `0` are the interface for which it is supposed to work. Could you change 23.7 to 1 and add an extra comment that any number will do? Thanks!

Reply:
The example directly preceding this illustrates the binary integer use case. I moved that example first, expecting that it's the intended case.

Given that a dictionary can hold anything, this second set of examples is meant to show how it will respond to a range of objects (i also included a Symbol). 

Should I still modify it?

Reply:
no, it's actually ok. I just have the uneasy feeling that we promote non-standard use if it is not explictly mentioned.

Reply:
ah ok, I see also that the diff obscured the example reordering, so that may have been misleading